src/antsibull/cli/antsibull_build.py

```diff
@@ -266,12 +272,20 @@ def parse_args(program_name: str, args: list[str]) -> argparse.Namespace:
                             help='Allow prereleases of collections to be included in the build'
                             ' file')
 
-    subparsers.add_parser('prepare',
-                          parents=[
-                              build_write_data_parser, build_step_parser, feature_freeze_parser,
-                              galaxy_file_parser,
-                          ],
-                          description='Collect dependencies for an Ansible release')
+    prepare_parser = subparsers.add_parser(
+        'prepare',
+        parents=[
+            build_write_data_parser,
+            build_step_parser,
+            feature_freeze_parser,
+            galaxy_file_parser,
+        ],
+        description='Collect dependencies for an Ansible release',
+    )
+    prepare_parser.add_argument(
+        '--tags-file', action='store_true', default=None,
+        help='Whether to include a tags data file in --dest-data-dir'
```

Comment:
Why use a boolean instead of a string?

Reply:
When `--tags-file` is passed, args.tags_file is set to f'{DEFAULT_FILE_BASE}-{args.ansible_version}-tags.yaml'. Otherwise, it's None. What are you proposing instead?

Reply:
What we discussed in #468: that it specifies the filename (eventually with a default, not sure when we should switch that on).

Reply:
Ah, you mean we'll eventually define a default string value to make this option enabled by default?

Reply:
Yes, I think that would be best.

Reply:
I've adjusted this to accept an optional argument as we discussed on IRC. I think this is a good middle ground :).